resources/make_test_resources.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Can we reuse this in the deps-pipeline so that we do not have duplicated commands?

Reply:
Synced offline. The disk step requires `privileged: true`, so we can't reuse this script.